src/components/Table/ListTable/TableAction/QuickFilter.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <div v-show="isExpand">
-    <div v-if="filters || summary" :class="isExpand ? 'expand': 'shrink' " class="quick-filter">
+    <div
+      v-if="(filters && filters.length > 0) || (summary && summary.length > 0)"
+      :class="isExpand ? 'expand': 'shrink' "
+      class="quick-filter"
+    >
       <div v-show="isExpand" class="quick-filter-wrap">
         <div v-if="filters" class="quick-filter-zone">
           <div v-for="category in iFilters" :key="category.label" class="item-zone">
```

Comment:
The code you provided seems to be written in a Vue template syntax, possibly for rendering components in a web application. There don't appear to be any apparent errors, oddities or optimization suggestions related to this particular snippet. I would recommend continuing to maintain the existing implementation, but if you need help with specific optimizations, further review, or general tips on better structure using Vue.js, feel free to ask!